### ISSUE TITLE
Can't share files after upgrading from OC 8.2.9 to NC 11.0

### ISSUE THREAD

**Issue by doonya** [nextcloud/server#2899]
After an upgrading from oc to NC 11.0 filesharing via Web si not working 
Ive installed à New NC 11 on another subdomain
With a New db and imported my users to this db it work but when I change the db and the data folder in config.php to my data and db the same problem reproduce

**Comment by MariusBluem** [comment]
Please fill out the **[issue template](https://github.com/nextcloud/server/blob/master/issue_template.md)** 😅 
Only with this information we are able to reproduce - and help you best 💪

**Comment by MariusBluem** [comment]
Please report issues only once ... you've already seen another report out there which is exactly about the same and [commented there](https://github.com/nextcloud/server/issues/2889#issuecomment-269843001) - so there is no need for opening another issue about the same topic 😅 Correct me if I am wrong 😁 

We continiue this here: https://github.com/nextcloud/server/issues/2889#issuecomment-269843001

Closing here now 😉
